packages/state-transition/src/cache/epochCache.ts

```diff
@@ -1151,6 +1175,34 @@ export class EpochCache {
   isPostElectra(): boolean {
     return this.epoch >= this.config.ELECTRA_FORK_EPOCH;
   }
+
+  getPTC(slot: Slot): ValidatorIndex[] {
+    const epoch = computeEpochAtSlot(slot);
+
+    if (epoch < this.config.GLOAS_FORK_EPOCH) {
+      throw new Error("PTC is not available before GLOAS fork");
```

Comment:
usually fork is all lowercase in error messages

```suggestion
      throw new Error("PTC is not available before gloas fork");
```